test/fixtures/buildDependencies/node_modules/dependency-with-exports/main.js

```diff
@@ -1,4 +1,2 @@
 require("dep-with-exports");
 require("dep-with-exports/sub");
-require("dep-without-package.json/main-entry");
-require("dep-without-package.json/sub-entry");
```

Comment:
hmm.. this doesn't work anymore... I guess we should add some kind of fallback for that to no regress on that. (should be added to `main`)

Reply:
Seems like the test wasn't working as intended, as it didn't use managedPaths for the node_modules folder...

Reply:
> Seems like the test wasn't working as intended, as it didn't use managedPaths for the node_modules folder...

we can fix test case in `main` (provide `managedPaths` in `webpack.config`)

Reply:
yep

Reply:
fixed by #14735